Help section (#136)
msupply-foundation/msupply_docs: content/docs/help/contact_us.md

+++

> title = "Contact Us"

Reviewer: Would also call this file/page `Help` - then we can include a section about the link to the docs here too

description = "Contact mSupply Foundation"
date = 2025-01-06T11:45:00+00:00
updated = 2025-01-06T11:45:00+00:00
template = "docs/page.html"
sort_by = "weight"
weight = 1
draft = false

[extra]
toc = true
top = false
+++

The contact form allows you to reach out to us with any feedback or support requests.

## Viewing the contact form

To view the contact form, go to `Help` in the lower section of the navigation panel:

> ![Help: nav](/docs/help/images/help_nav.png)

Reviewer: Would put a `This will take you to the Help page` with a full page screenshot after this

> You will see the contact form and are required to choose the reason for contacting us. Clicking on the dropdown will show you the type of requests:

Reviewer: NIT, maybe friendlier to say something like 

```suggestion
Let us know why you are getting in touch my clicking the dropdown:
```

![Help: reason](/docs/help/images/contact_form_reason.png)

Fill out the rest of the form accordingly:

![Help: body](/docs/help/images/contact_form_body.png)

Once the form has been filled out, the send button can now be clicked. Proceed to click send once satisfied with your message to us:

> ![Help: send](/docs/help/images/contact_form_send.png)

Reviewer: Would add something here like "The mSupply Support team will receive your message, and be in touch soon to respond to support requests."
